server/schedulers/evict_leader.go

```diff
@@ -262,10 +269,15 @@ func uniqueAppendOperator(dst []*operator.Operator, src ...*operator.Operator) [
 	return dst
 }
 
-func scheduleEvictLeaderBatch(name, typ string, cluster schedule.Cluster, storeRanges map[uint64][]core.KeyRange, batchSize int) []*operator.Operator {
+type evictLeaderStoresConf interface {
```

Comment:
may we don't need this interface

Reply:
We need this interface since we need to use the function `scheduleEvictLeaderBatch` both for `evictLeaderScheduler` and `evictSlowStoreScheduler`.